File: lesunny/026_자연수뒤집어배열로만들기.java
import java.util.ArrayList;

class Solution {
    public int[] solution(long n) {
        ArrayList<Long> list_answer = new ArrayList<>();

        while(n!=0){
            list_answer.add(n%10);
            n /= 10;
        }
        
        int[] answer = new int[list_answer.size()];
        for(int i=0; i<list_answer.size();i++){
            answer[i] = list_answer.get(i).intValue();
        }
        
        return answer;
    }
}

File: jiheekim/026_자연수 뒤집어 배열로 만들기.cpp
#include <string>
#include <vector>

using namespace std;

vector<int> solution(long long n) {
    vector<int> answer;
    int i=0;
	while (n != 0) {
		i = n % 10;
		answer.push_back(i);
		n = n / 10;
	}
    return answer;
}
